### ISSUE TITLE
Add proxy URL for Brazil's CAPES Periódicos

### ISSUE THREAD

**Issue by gersonjferreira** [tom5760/ezproxy-db#109]
### Name

CAPES Periódicos

### URL

http://novo.periodicos.capes.gov.br/?option=com_pezproxy&controller=auth&view=pezproxyauth&url=$@

### Country

Brazil

**Comment by gersonjferreira** [comment]
I've developed, with Bruno Messias, a Chrome/Firefox extension to do this redirect here in Brazil (we need it even inside the university network). You can check here:

- https://chrome.google.com/webstore/detail/redirecionamento-capes-pe/lpfcdddcpijdfghkimmcpkmidafnljbo
- https://addons.mozilla.org/en-US/firefox/addon/redir-capes-periodicos/
- https://openuserjs.org/scripts/gersonjferreira/Redirecionamento_CAPES-Peri%C3%B3dicos
- https://subgnano.gitlab.io/projects.html

But your extension does pretty much the same thing. We have tried to implement an auto-redirect feature that fails sometimes. It has a list of websites that are accessible via the proxy, and you and/remove to white/black lists.

Since this automatic feature fails and we don't have time to maintain it, I will recommend people to start using your extension instead. I'll do the announcements once this new URL is added to the list.

**Comment by tom5760** [comment]
The site has been updated.  Thanks @gersonjferreira!

I also got a similar feature request to support auto-redirect, in tom5760/chrome-ezproxy#9, but similarly haven't had much time lately to try to implement it.  I think it will be tricky without expanding the permissions a lot.

Anyway, thanks for the recommendation!  Hopefully will be able to do some sort of auto-redirect soon...

**Comment by gersonjferreira** [comment]
Thanks!

The auto-redirect works on many journals, but sometimes it fails badly. For instance, if we try to access prb.aps.org to submit a paper, it tries to redirect the login page and breaks it. Similarly, it breaks the login on web-of-science. We can put web-of-science on a blacklist, but for prb.aps.org the same domain is used for the system and for the papers.

One problem we had is that we couldn't find a way identify a fail redirect and move back automatically. Also, sometimes a bad redirect cannot be reversed with the browser "go back" button.

So... be careful with auto-redirect...

**Comment by tom5760** [comment]
> So... be careful with auto-redirect...

Ah!  I hadn't even thought of those kinds of cases.  Yeah, might be even more complicated than I thought!

**Comment by gersonjferreira** [comment]
Yes, it's tricky!

Here in Brazil, CAPES decided to implement EZProxy without a warning. So, out of nowhere we couldn't download papers anymore. We still have to use the proxy even inside the university! I've released a first version my extension a couple of days after this, and the number of users exploded quickly!

A couple of weeks later I actually got a call from web-of-science, because the number of users complaining that their web page was broken was huge! So I had to blacklist web-of-science from the auto-redirect.
